### PR TITLE
Common code changes before integrating Mynewt

### PR DESCRIPTION
**Pull request by michal-narajowski**

### CODE REVIEW

File: pybtp/btp.py [hunk 1]
```diff
@@ -136,7 +136,9 @@
     "get_attrs": (defs.BTP_SERVICE_ID_GATT, defs.GATT_GET_ATTRIBUTES,
                   CONTROLLER_INDEX),
     "get_attr_val": (defs.BTP_SERVICE_ID_GATT,
-                     defs.GATT_GET_ATTRIBUTE_VALUE, CONTROLLER_INDEX)
+                     defs.GATT_GET_ATTRIBUTE_VALUE, CONTROLLER_INDEX),
+    "change_database": (defs.BTP_SERVICE_ID_GATT,
```
Comment (MariuszSkamra) [inline pybtp/btp.py]: BTP spec shall be updated as well

File: pybtp/btp.py [hunk 2]
```diff
@@ -110,6 +110,8 @@
     "passkey_entry_rsp": (defs.BTP_SERVICE_ID_GAP,
                           defs.GAP_PASSKEY_ENTRY,
                           CONTROLLER_INDEX),
+    "start_direct_adv": (defs.BTP_SERVICE_ID_GAP, defs.GAP_START_DIRECT_ADV,
```
Comment (MariuszSkamra) [inline pybtp/btp.py]: ditto, BTP spec shall be updated as well

File: pybtp/btp.py [hunk 3]
```diff
@@ -166,6 +166,8 @@
     "write": (defs.BTP_SERVICE_ID_GATT, defs.GATT_WRITE, CONTROLLER_INDEX),
     "write_long": (defs.BTP_SERVICE_ID_GATT, defs.GATT_WRITE_LONG,
                    CONTROLLER_INDEX),
+    "write_reliable": (defs.BTP_SERVICE_ID_GATT, defs.GATT_WRITE_RELIABLE,
```
Comment (MariuszSkamra) [inline pybtp/btp.py]: BTP spec shall be updated as well

File: pybtp/btp.py (hunk 2 above)
Comment (michal-narajowski) [reply pybtp/btp.py]: There is a Start Directed Advertising command in the BTP spec already.

File: pybtp/btp.py (hunk 3 above)
Comment (michal-narajowski) [reply pybtp/btp.py]: There is a Reliable Write command in the BTP spec already.

File: pybtp/btp.py (hunk 1 above)
Comment (michal-narajowski) [reply pybtp/btp.py]: I updated the commit.

File: pybtp/btp.py (hunk 3 above)
Comment (MariuszSkamra) [reply pybtp/btp.py]: OK :)

File: pybtp/btp.py (hunk 2 above)
Comment (MariuszSkamra) [reply pybtp/btp.py]: OK :)

**Comment by MariuszSkamra** [comment]
LGTM

**Pull request merged**